test/puppetlabs/trapperkeeper/services/status/status_core_test.clj

```diff
@@ -110,10 +110,18 @@
       (is (= #{:jvm-metrics} (ks/keyset (get-in status [:status :experimental]))))
       (let [jvm-metrics (get-in status [:status :experimental :jvm-metrics])]
         (is (= #{:heap-memory :non-heap-memory
+                 :file-descriptors :gc-stats
                  :up-time-ms :start-time-ms} (ks/keyset jvm-metrics)))
         (is (= #{:committed :init :max :used} (ks/keyset (:heap-memory jvm-metrics))))
         (is (= #{:committed :init :max :used} (ks/keyset (:non-heap-memory jvm-metrics))))
         (is (every? #(< 0 %) (vals (:heap-memory jvm-metrics))))
         (is (every? #(or (< 0 %) (= -1 %)) (vals (:non-heap-memory jvm-metrics))))
+        (is (= #{:max :used} (ks/keyset (:file-descriptors jvm-metrics))))
+        (is (every? #(< 0 %) (vals (:file-descriptors jvm-metrics))))
+        (is (every? #(= #{:count :total-time-ms} (ks/keyset %))
+                    (vals (:gc-stats jvm-metrics))))
+        (is (every? #(<= 0 %) ;; Possible that no major collections occurred.
+                    (mapcat #(vals %)
```

Comment:
A small clojure thing - this doesn't need to be `(mapcat #(vals %)... ` - it can just be `(mapcat vals...` functions that take a single arg can be passed to mapping functions without parens. I think basically `map` and friends take as their first argument a function that takes a single argument. Wrapping things in `#()` or `(fn [x])` creates a function that takes one arg, but isn't necessary if you are using an already existing function that takes a single arg (this also includes keywords, since keywords are functions - e.g. `(map :a [{:a 1} {:a 2} {:a 3}]) => (1 2 3)`)

(Not something that needs to be fixed in this PR - just an FYI).